src/nvim/types.h

```diff
@@ -13,4 +13,5 @@ typedef unsigned char char_u;
 // Can hold one decoded UTF-8 character.
 typedef uint32_t u8char_T;
 
+typedef struct expand expand_T;
```

Comment:
Ok, I kind of forgot about that. In the very beginning of the project there was enthusiasm for moving everything to types.h, away from, say, vim.h and globals.h. It wasn't really clear why, though. If helpers.lua depends on it, it's fine as you have it. Sorry for the noise.